src/main/java/com/kornello/controller/NasaPictureController.java

```diff
@@ -0,0 +1,30 @@
+package com.kornello.controller;
+
+import com.kornello.service.NasaPictureService;
+import com.kornello.model.Picture;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.HttpStatus.OK;
+
+@RestController
+@RequestMapping("/api")
+@RequiredArgsConstructor
+public class NasaPictureController {
+    private final NasaPictureService service;
+
+    final HttpHeaders headers = new HttpHeaders() {{
+        add("content-type","image/jpeg");
+    }};
+
+    @GetMapping({"/mars/pictures/largest/{sol}", "/mars/pictures/largest/{sol}/{camera}"})
+    public ResponseEntity<?> getLargestPicture(@PathVariable String sol, @PathVariable(required = false) String cam) {
```

Comment:
Please use <byte[]> instead of wildcard. It's not a good practice to use wildcards in the return type